### DISCUSSION TITLE
Ingress 方式下如何给路由配置插件了？

### DISCUSSION THREAD

**Discussion by jaggerwang** [higress-group/higress#1086] [Q&A] [answered]
在 Web 控制台，可以给某个路由配置策略，也就是添加插件，比如认证或限流，但 Ingress 方式下了？没看到相关文档，只看到了各个插件的配置参数说明，但怎么给 Ingress 里的某个 Rule 配置一个或多个插件了？

**Comment by CH3CHO** [comment]
是说给路由配插件吗？点击路由左右侧的策略链接就可以了。

![image](https://github.com/alibaba/higress/assets/2909796/c6c3dcdb-a69e-4f94-8b06-ab5f2b1d6c8d)

**Reply by jaggerwang** [reply]
Ingress 方式，界面方式限制很多。

**Reply by CH3CHO** [reply]
一样的。用 WasmPlugin CRD 来配置就行。你可以先在 Console 里配置一下，然后 kubectl get wasmplugins -n higress-system，照葫芦画瓢就行了。

**Reply by jaggerwang** [reply]
有没有官方的相关文档链接。

**Reply by CH3CHO** [reply, marked as answer]
https://higress.io/zh-cn/docs/plugins/intro

**Reply by jaggerwang** [reply]
OK，谢谢！
